File: app/src/main/java/com/example/wellbees_project/ApiUtils.java
package com.example.wellbees_project;

public class ApiUtils {

    public static final String BASE_URL = "https://newsapi.org";

    public static SourcesDAOInterface getSourcesDaoInterface(){
        return RetrofitClient.getClient(BASE_URL).create(SourcesDAOInterface.class);
    }

}

File: app/src/main/java/com/example/wellbees_project/ApiUtils.kt
package com.example.wellbees_project

object ApiUtils {
    const val BASE_URL = "https://newsapi.org"
    val sourcesDaoInterface: SourcesDAOInterface
        get() = RetrofitClient.getClient(BASE_URL)!!.create(SourcesDAOInterface::class.java)
}